lepton/utils.go

```diff
@@ -0,0 +1,11 @@
+package lepton
+
+import (
+	"fmt"
+	"os"
+)
+
+func exitWithError(errs string) {
```

Comment:
this is a dupe but it's relatively innocuous - i suppose there is room for a utils pkg or something in the future

Reply:
There's a utils in cmd package but I thought lepton package shouldn't depend on cmd. As you said definately in future if we need more utilities like this, I will create separate utils package and refactor code accordingly.